components/eam/src/chemistry/mozart/lin_strat_chem.F90

```diff
@@ -183,26 +185,36 @@ subroutine lin_strat_chem_inti(phys_state)
 
     ! define additional output
 
-    call addfld( 'LINOZ_DO3'    , (/ 'lev' /), 'A', '/s'     , 'ozone vmr tendency by linearized ozone chemistry'   )
-    call addfld( 'LINOZ_DO3_PSC', (/ 'lev' /), 'A', '/s'     , 'ozone vmr loss by PSCs using Carille et al. (1990)' )
+    call addfld( 'LINOZ_DO3LNZ'    , (/ 'lev' /), 'A', '/s'     , 'ozone vmr tendency by linearized ozone chemistry'   )
+    call addfld( 'LINOZ_DO3LNZ_PSC', (/ 'lev' /), 'A', '/s'     , 'ozone vmr loss by PSCs using Carille et al. (1990)' )
+    call addfld( 'LINOZ_2DDO3LNZ'    , horiz_only, 'A', 'DU/s'     , 'ozone 2D DU tendency by linearized ozone chemistry'   )
+    call addfld( 'LINOZ_2DDO3LNZ_PSC', horiz_only, 'A', 'DU/s'     , 'ozone 2D DU loss by PSCs using Carille et al. (1990)' )
     call addfld( 'LINOZ_SSO3'   , (/ 'lev' /), 'A', 'kg'     , 'steady state ozone in LINOZ'                        )
     call addfld( 'LINOZ_O3COL'  , (/ 'lev' /), 'A', 'DU'     , 'ozone column above'                                 )
     call addfld( 'LINOZ_O3CLIM' , (/ 'lev' /), 'A', 'mol/mol', 'climatology of ozone in LINOZ'                      )
     call addfld( 'LINOZ_SZA'    ,    horiz_only, 'A', 'degrees', 'solar zenith angle in LINOZ'                      )
     call addfld( 'LINOZ_O3SFCSINK',  horiz_only, 'A', 'Tg/yr/m2'   ,   'surface o3lnz sink in LINOZ with an e-fold to a fixed concentration' )
+    call addfld( 'LINOZ_DO3'    , (/ 'lev' /), 'A', '/s'     , 'ozone vmr tendency by linearized ozone chemistry'   )
+    call addfld( 'LINOZ_DO3_PSC', (/ 'lev' /), 'A', '/s'     , 'ozone vmr loss by PSCs using Carille et al. (1990)' )
+    call addfld( 'LINOZ_2DDO3'  , horiz_only, 'A', 'DU/s'     , 'ozone 2D DU tendency by linearized ozone chemistry'   )
+    call addfld( 'LINOZ_2DDO3_PSC', horiz_only, 'A', 'DU/s'     , 'ozone 2D DU loss by PSCs using Carille et al. (1990)' )
```

Comment:
Similar to above. It is clearer to just use the tracer name "O3" instead of ozone.

Reply:
Changed.